Section_II/competition_and_trophies.tex

```diff
@@ -19,6 +19,7 @@
 from front and back (KidSize).
 The teams must be able to demonstrate at least one successful stand-up action
 from each side during the robot inspection.
+\added{The teams must also be able to demonstrate that the robot is capable of connecting to the Wi-Fi network.}
```

Comment:
Which one? One of the field networks of their choice? How do they demonstrate that?